sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/Sample.java

```diff
@@ -36,11 +36,11 @@ public void sample() throws IOException {
             .buildClient();
```

Comment:
Random aside.. shouldn't this sample be in.. samples? I'm a bit confused.

Reply:
Yeah, you're right! I checked Alan. This doesn't have good samples either to justify moving to samples. So, this should just be deleted.